episodes/chapter6.md

```diff
@@ -249,10 +202,11 @@ In the printed message, you will see not only the path of the configuration file
 
 ::::::::::::::::::::::::::::::::::::: challenge
 
-## Challenge 2: Create a new node configuration
+## Challenge 1: Create a new node configuration
 
 1. Create a new node configuration using the `v6 node new` command.
 2. Find the path to the configuration file using the `v6 node files` command. Open the configuration file with a text editor and check the configuration options. Are they correct?
+3. Add some missing options to your configuration file. You can find all options in the [vantage6 documentation](https://docs.vantage6.ai/en/main/node/configure.html#all-configuration-options).
```

Comment:
Maybe it would be good to specify which options these should be, as adding some options may lead to weird behaviour (e.g. if they add `allowed_algorithms` some algorithms later in the course may not be executed). And ideally it would be something they get reminded of later.

Reply:
Good catch. I'm afraid the participants will be blocked by what values to add for the missing options. So I would ask them to add a new database in a different format, change the encryption option, and find the missing options.

Reply:
That sounds good! Be sure to say that they should also change the encryption option back in that case